python/coinbase-agentkit/tests/action_providers/desk/conftest.py

```
@@ -0,0 +1,26 @@
+"""Test fixtures for DESK action provider tests."""
+
+from unittest.mock import Mock
```

Ruff (0.8.2)

`unittest.mock.Mock` imported but unused

Remove unused import: `unittest.mock.Mock`

(F401)